pkg/bosh/converter/container_factory.go

```diff
@@ -515,6 +518,15 @@ func bpmProcessContainer(
 		workdir = filepath.Join(VolumeJobsDirMountPath, jobName)
 	}
 	command, args := generateBPMCommand(&process, postStart)
+	limits := corev1.ResourceList{}
+	if process.Limits.Memory != "" {
+		quantity, err := resource.ParseQuantity(process.Limits.Memory)
+		if err != nil {
+			log.Errorf(context.TODO(), "Error parsing %s: %v", process.Limits.Memory, err)
+		} else {
+			limits[corev1.ResourceMemory] = quantity
+		}
+	}
```

Comment:
Just wondering, why the `limits` don't come from where the `requests` come from.

Reply:
The `Limits` structure in the `BPMProcess` was available before our change for the `Requests`. Therefore, I decided to use the data provided by `BPMProcess.Limits`